pennylane/ops/op_math/controlled.py

```diff
@@ -395,6 +397,16 @@ def __new__(cls, base, *_, **__):
             return object.__new__(ControlledOp)
         return object.__new__(Controlled)
 
+    # pylint: disable=arguments-differ
+    @classmethod
+    def _primitive_bind_call(
+        cls, base, control_wires, control_values=None, work_wires=None, id=None
+    ):
+        control_wires = Wires(control_wires)
+        return cls._primitive.bind(
+            base, *control_wires, control_values=control_values, work_wires=work_wires
```

Comment:
Should `work_wires` and `control_values` not be dynamic?

Reply:
Potentially?  I created a story in the epic to come back to that question.

Reply:
Great :)